src/components/Dashboard/dashboard.module.scss

```diff
@@ -0,0 +1,77 @@
+.dashboard { 
+    border-left: #dad8d8 solid 2px;
```

Comment:
check if ```var(--seed-border-light)``` works for your border colors

If not create a new CSS variable in app.scss instead of hardcoding hex values into your stylesheet 👍